app/src/main/java/com/example/lapnguyen/weather_3/screen/mainscreen/MainActivity.java

```diff
@@ -1,6 +1,7 @@
 package com.example.lapnguyen.weather_3.screen.mainscreen;
 
 import android.annotation.SuppressLint;
+import android.location.Location;
```

Comment:
ERROR: Unused import: android.location.Location. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports